cmd/site/config.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"os"
+
+	"github.com/golang-insiders/site/internal/data"
+	"github.com/joho/godotenv"
+)
+
+type config struct {
+	port string
+	env  string
+	db   data.PostgresConfig
+}
+
+func loadConfig() (config, error) {
+	var cfg config
+	err := godotenv.Load()
+	if err != nil {
+		return cfg, err
+	}
+
+	cfg.port = os.Getenv("PORT")
```

Comment:
Do we need to check these are not empty?

Reply:
I feel we should check